spec/javascripts/components/gtm-form-listener-spec.js

```diff
@@ -0,0 +1,33 @@
+var $ = window.jQuery
+
+describe('GTM dataLayer messages for radio button submissions', function () {
+  'use strict'
+
+  var form
+  var dataLayer
+  var gtmFormListener
+
+  beforeEach(function () {
+    form = $(`
```

Comment:
I think @benthorner did some work stripping out jQuery from all our tests so if we could use DOM methods that'd be consistent

Reply:
Fixed in 8cfc33a092bf9321f17a35c744e9120106ec3965